sql/catalyst/pom.xml

```diff
@@ -116,6 +116,10 @@
       <artifactId>datasketches-java</artifactId>
       <version>3.3.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
Isn't this already included transitively?

Reply:
Yes, the addition can be removed. I just committed the change. It was likely added in the wrong place because I was porting my changes from the 3.4 branch where I worked originally.